modules/s2/redshift/lib/version.py

```diff
@@ -0,0 +1 @@
+__version__ = '1.0.0'
```

Comment:
Use `_version.py` which is more standard

Reply:
Ok! I thought `version.py` was more standard looking at: https://stackoverflow.com/a/16084844/8159847

Reply:
I have seen more `_version` in other projects, like numpy, pandas, geopandas